RealEstate/apps/core/templates/core/houseEval.html

```diff
@@ -44,9 +44,9 @@
 					<div class="col-sm-6 vcenter rating">
 						<div class="inner-rating" >
 							<center>
-								<span class="help-text" style="padding-right: 15px">Bad </span>
-									<input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min="1" data-slider-max="5" data-slider-step="1" data-slider-value="{% if not score %}3{% else %}{{ score }}{% endif %}"/>
-								<span class="help-text" style="padding-left: 15px">Good</span>
+                                <span class="help-text" style="padding-right: 15px">{{min_choice}}</span>
+                                <input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min={{min_score}} data-slider-max={{max_score}} data-slider-step="1" data-slider-value="{% if not score %}{{default_score}}{% else %}{{ score }}{% endif %}"/>
+                                <span class="help-text" style="padding-left: 15px">{{max_choice}}</span>
```

Comment:
Something's wonky with the indentation here. I'm sure that's an issue with you working on it after my lame text editor. But can we try to get it back?